policyengine_us/parameters/gov/states/id/tax/income/subtractions/subtractions.yaml

```diff
@@ -6,6 +6,7 @@ values:
     - taxable_social_security # (5)
     - military_service_income # (4)
     - workers_compensation # (13)
+    - id_aged_or_disabled_deduction #(8)
```

Comment:
in the tax form, it is mentioned in subtraction, but it is called a deduction

Reply:
I didn't change the "deductions" in the description and labels. I only moved those files to "subtraction" folder

Reply:
sounds good thanks

Reply:
For posterity: the legal code might suggest it would act as a deduction:
>An additional deduction **from taxable income** shall be allowed

However, [form 40](https://tax.idaho.gov/wp-content/uploads/forms/EFO00089/EFO00089_12-30-2022.pdf#page=1) indicates that it subtracts from AGI, making this categorization both correct for Form 39R and aligning it to general notions of subtractions vs deductions.